evaluation/evaluation_engine.py

```diff
@@ -55,14 +59,105 @@ def execute_model(implementation, task_type, locale="en", model=None, dataset=No
               f"the right place to do it would to add a new function in evaluate/evaluation_engine.py "
               f"and call it from execute_model. That's it!")
 
+def conll_tokens_to_sentence(token_list):
```

Comment:
It will be better if we have an evaluation_util.py file and move all these util functions in that file.
This file should only have evaluate functions corresponding to different tasks.